packages/firestore/test/integration/api/query.test.ts

```diff
@@ -932,6 +955,18 @@ apiDescribe('Queries', (persistence: boolean) => {
         .where('array', 'array-contains-any', [{ a: 42 }])
         .get();
       expect(toDataArray(snapshot2)).to.deep.equal([{ array: [{ a: 42 }] }]);
+
+      // With null.
+      const snapshot3 = await coll
+        .where('zip', 'array-contains-any', [null])
```

Comment:
Do we need a test that shows that "array-contains-any" with null/nan and something else still finds the something else?

Reply:
done.